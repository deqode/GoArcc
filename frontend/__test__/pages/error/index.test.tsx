import renderer from 'react-test-renderer'

import ErrorPage from '../../../pages/error/index'

jest.mock('next/router', () => ({
  useRouter: jest.fn().mockImplementation(() => ({
    query: {
      message: 'Network Error',
    },
  })),
}))

describe('error page tests', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<ErrorPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
