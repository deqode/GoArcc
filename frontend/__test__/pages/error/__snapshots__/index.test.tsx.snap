// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error page tests should match the snapshot 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-around"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-lg-4 MuiGrid-grid-xl-4"
  >
    <div
      className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation0 MuiPaper-rounded"
    >
      <h1
        className="MuiTypography-root MuiTypography-h3 MuiTypography-colorPrimary"
      >
        Alfred has encountered with
      </h1>
      <br />
      <h5
        className="MuiTypography-root MuiTypography-h5 MuiTypography-colorSecondary"
      />
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-lg-4 MuiGrid-grid-xl-4"
  >
    <div
      className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation0 MuiPaper-rounded"
    >
      <p
        className="MuiTypography-root makeStyles-text-3 MuiTypography-body1 MuiTypography-displayInline"
      >
        Network Error
      </p>
    </div>
  </div>
</div>
`;
